ament_pep257/ament_pep257/main.py

```diff
@@ -45,11 +45,14 @@ def main(argv=sys.argv[1:]):
     parser.add_argument(
         '--ignore',
         nargs='*',
-        default=[
-            'D100', 'D101', 'D102', 'D103', 'D104', 'D105', 'D106', 'D107',
-            'D203', 'D212', 'D404',
-        ],
+        default=['D1'],
```

Comment:
Given the point of this PR, it's worth mentioning that leaving this as the default still means this isn't exactly pep257.

Reply:
0b9856b added a flag and a warning about it being required in the future so as to not break current tests